rmgpy/chemkin.pyx

```diff
@@ -1567,13 +1588,15 @@ def writeReactionString(reaction, javaLibrary = False):
     
 ################################################################################
 
-def writeKineticsEntry(reaction, speciesList, verbose = True, javaLibrary = False, commented=False):
+def writeKineticsEntry(reaction, speciesList, verbose=True, javaLibrary=False, commented=False, use_label=False):
     """
     Return a string representation of the reaction as used in a Chemkin
     file. Use `verbose = True` to turn on kinetics comments.
     Use `commented = True` to comment out the entire reaction.
     Use javaLibrary = True in order to generate a kinetics entry suitable
-    for an RMG-Java kinetics library.
+    for an RMG-Java kinetics library. `use_label` flag indicates that
+    the reaction string should use the species.label instead of the default RMG
+    species identifier.
```

Comment:
I would consider the species label the default RMG species identifier. Suggestion:
```
Use `use_label=True` to force use of the default RMG species label instead 
of a ensuring a Chemkin compatible species identifier in the reaction string.
```

Reply:
fixed for all three instances